docs/basics/beginners-contributing-guide.md

```diff
@@ -0,0 +1,119 @@
+upstream        https://github.com/selfhostedshow/wiki (push)
+```
+
+#### 1.2 Start the Wiki Docker Container
+This section will start the wiki docker container in the local environment.
+
+__Note__: The below path may be different, depending on where the wiki was cloned to.
+
+__Note__: -f will define a docker-compose file and -d ("detached" mode) will start the wiki it in the background
+
+0. Type `sudo docker-compose -f ~/PATHTOWIKI/wiki/docker-compose.yml up -d` this will start the wiki in docker.
+
+0. Type `docker ps` to verify that the wiki container is running.
+
+0. Navigate to `http://DockerIP:8000` the "dev" branch wiki website will then open.
+
+0. Now edit the wiki site in your editor of choice and when you **save** the file, the site will be updated with the changes that are made enabling real time (more or less) verifying what the site will look like with the changes.
+
+#### 1.3 Merging an local repository into your Fork
+This Section will "merge" the local changes to your fork (your github repo that was made above in [section 1.1](https://github.com/selfhostedshow/wiki/tree/dev/docs\basics/beginners-contributing-guide/#11-fork-clone-the-self-hosted-show-wiki))
+
+0. Type `git add YOURFILENAME1.md YOURFILENAME2.md` to add the changed files to your commit.
+
+0. Type `git commit -m 'updated x,y,z'` to commit the changes to your fork with comments.
+
+0. Type  `git push origin dev` to push your commit to your fork.
+
+0. Type your username for github:
+
+0. Type your password for github:
+
+#### 1.3 Make a New Pull Request on the GitHub Website
+This section will create a new pull request from your Github repo to the [Upstream](https://github.com/selfhostedshow/wiki) wiki site.
+
+0. Navigate to the Github [wiki](https://github.com/selfhostedshow/wiki) web interface.
+
+0. Click ![newpulrequest](images/newpullrequrest.PNG) in the top(ish) left, verify the the branch is set to "dev". 
+
+0. Under Compare changes click "compare across forks" link, verify the **base** repository is the [upstream](https://github.com/selfhostedshow/wiki) site and the head repository is your repo.
+
+0. The changes that were made should come automatically, make the comment succinct and meaningful.
+
+0. Submit a pull request by clicking ![createpullrequest](images/createpullrequest.PNG).
+
+0. The commit has now been added to the pull request the Wiki-Admins will review and if approved will get pushed to the master branch for publishing.
+
+### 2 Syncing a Fork
+This Section will "pull" from the [upstream](https://github.com/selfhostedshow/wiki) repository to your local repository. **This should be done every time before you make any changes to your local wiki in order to make sure everything is in sync**.
+
+-  Verify you are in the wiki directory.
+
+- Type `git fetch upstream` to fetch the branches and their respective commits from the upstream repository
+
+- Verify the "dev" branch is checked out type `git checkout -b dev`
```

Comment:
The same. Should be `git checkout dev`, because this one will give a fatal error